### ISSUE TITLE
Bug in 3.6.0? Win10: "docker builderx create" creates an inactive instance

### ISSUE THREAD

**Issue by ChrisKuBa** [docker/buildx#754]
Hi,

after getting some problems with Docker Desktop for windows, I try to reinstall and restart Docker Desktop.

The main problem is, that the builder instance doesnt run anymore.

All works fine before the crash of Docker Desktop. Also with buildx.

Currently I build ever "arch" separately with the default build instance.

Hope someone can help me with the error
**Error response from daemon: invalid mount config for type "volume": invalid mount path: 'C:/ProgramData/buildkitd/.buildstate' mount path must be absolute**

Thx

Some more infos:

Docker Desktop: 3.6.0 (67351)
Latest windows updates.
WSL2 is active

```
wsl -l -v
  NAME                   STATE           VERSION
* docker-desktop         Running         2
  docker-desktop-data    Running         2

docker buildx create --use --name chriskuba

docker buildx ls
NAME/NODE       DRIVER/ENDPOINT                STATUS   PLATFORMS
chriskuba *     docker-container
  chriskuba0    npipe:////./pipe/docker_engine inactive
desktop-linux   docker
  desktop-linux desktop-linux                  running  linux/amd64, linux/arm64, linux/riscv64, linux/ppc64le, linux/s390x, linux/386, linux/arm/v7, linux/arm/v6
default         docker
  default       default                        running  linux/amd64, linux/arm64, linux/riscv64, linux/ppc64le, linux/s390x, linux/386, linux/arm/v7, linux/arm/v6

docker buildx inspect --bootstrap
[+] Building 0.0s (1/1) FINISHED
 => ERROR [internal] booting buildkit                                                                                                                                                                     4.3s
 => => pulling image moby/buildkit:buildx-stable-1                                                                                                                                                        4.2s
 => => creating container buildx_buildkit_chriskuba0                                                                                                                                                      0.1s
------
 > [internal] booting buildkit:
------
Name:   chriskuba
Driver: docker-container

Nodes:
Name:     chriskuba0
Endpoint: npipe:////./pipe/docker_engine
Error:    Error response from daemon: invalid mount config for type "volume": invalid mount path: 'C:/ProgramData/buildkitd/.buildstate' mount path must be absolute
```

**Comment by ChrisKuBa** [comment]
A simple **downgrade** to version **3.5.2** solved the issue. Using my own and existing build instance. without a problem.

**Comment by tonistiigi** [comment]
This was fixed in #751
